superset/models/cache.py

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from datetime import datetime
+
+from flask_appbuilder import Model
+from sqlalchemy import Column, DateTime, Integer, String
+
+
+class CacheKey(Model):  # pylint: disable=too-few-public-methods
+
+    """Stores cache key records for the superset visualization."""
+
+    __tablename__ = "cache_keys"
+    id = Column(Integer, primary_key=True)
+    cache_key = Column(String(32), nullable=False)
```

Comment:
This seems awfully short for an arbitrary cache key - are we certain they will only ever be 32 characters?

Reply:
changed to 256